ansible/playbook.yml

```
@@ -0,0 +1,77 @@
+- name: setup servers
+  hosts: webservers
+  remote_user: ec2-user 
+
+  vars_prompt:
+    - name: trello_token
+      prompt: "Enter your Trello API token"
+      private: yes 
+
+  tasks:
+    - name: Install Git
+      yum:
+        name: git
+        state: present
+      become: yes
+
+    - name: Install Python 3
+      yum:
+        name: python3
+        state: present
+      become: yes
```

Minor but worth noting that we can combine these first couple together if we like:
```suggestion
    - name: Install Git & Python
      yum:
        name: 
          - git
          - python3
        state: present
      become: yes
```
(And include pip if we want too)